<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1926</width>
    <height>1022</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>141</width>
     <height>971</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string/>
      </property>
      <widget class="QListWidget" name="listWidget_cells">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>310</y>
         <width>111</width>
         <height>231</height>
        </rect>
       </property>
       <property name="currentRow">
        <number>-1</number>
       </property>
      </widget>
      <widget class="QListWidget" name="listWidget_sweeps">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>570</y>
         <width>111</width>
         <height>111</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_ct">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>71</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Groupby</string>
       </property>
      </widget>
      <widget class="QListWidget" name="listWidget_ct">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>111</width>
         <height>211</height>
        </rect>
       </property>
       <property name="currentRow">
        <number>0</number>
       </property>
       <item>
        <property name="text">
         <string>All</string>
        </property>
       </item>
      </widget>
      <widget class="QLabel" name="label_ct_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>290</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Cell list</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_ct_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>550</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Sweep list</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_cellNum">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>290</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>N=?</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_show_gene">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>750</y>
         <width>91</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Show gene exp.</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="textEdit">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>710</y>
         <width>111</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_ct_5">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>690</y>
         <width>91</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Gene</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_sweepNum">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>550</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>N=?</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox_groupBy">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>111</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QGroupBox" name="groupBox_filter">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>780</y>
         <width>120</width>
         <height>171</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="title">
        <string>Filters</string>
       </property>
       <widget class="QCheckBox" name="cb_hasMorph">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>60</y>
          <width>111</width>
          <height>17</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>9</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>has Morphorlogy</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_ct_8">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>80</y>
          <width>111</width>
          <height>20</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>8</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>Mapping rate cut-off</string>
        </property>
       </widget>
       <widget class="QSpinBox" name="spinBox_mappingrate">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>110</y>
          <width>31</width>
          <height>22</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
        <property name="value">
         <number>50</number>
        </property>
       </widget>
       <widget class="QCheckBox" name="checkBox_goodEphy">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>111</width>
          <height>17</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>&gt;=OK recording</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QSlider" name="horizontalSlider_mappingrate">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>110</y>
          <width>71</width>
          <height>22</height>
         </rect>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="value">
         <number>50</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="tickPosition">
         <enum>QSlider::TicksBelow</enum>
        </property>
        <property name="tickInterval">
         <number>10</number>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>10</y>
     <width>991</width>
     <height>971</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0" colspan="2">
     <widget class="QTabWidget" name="tabWidget_Transcitome">
      <property name="currentIndex">
       <number>-1</number>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget_Morphor">
      <property name="currentIndex">
       <number>-1</number>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QTabWidget" name="tabWidget_Ephy">
      <property name="currentIndex">
       <number>-1</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_ct_4">
   <property name="geometry">
    <rect>
     <x>1230</x>
     <y>0</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Ephy-meta</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget_EphyMetaData">
   <property name="geometry">
    <rect>
     <x>1160</x>
     <y>20</y>
     <width>241</width>
     <height>991</height>
    </rect>
   </property>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <column>
    <property name="text">
     <string>Feature</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Value</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="label_ct_7">
   <property name="geometry">
    <rect>
     <x>1450</x>
     <y>0</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Morphorlogy-meta</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget_morphorMetaData">
   <property name="geometry">
    <rect>
     <x>1410</x>
     <y>20</y>
     <width>241</width>
     <height>991</height>
    </rect>
   </property>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <column>
    <property name="text">
     <string>Feature</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Value</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="label_ct_6">
   <property name="geometry">
    <rect>
     <x>1700</x>
     <y>0</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Transcriptomicmeta</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget_TransMetaData">
   <property name="geometry">
    <rect>
     <x>1660</x>
     <y>20</y>
     <width>241</width>
     <height>991</height>
    </rect>
   </property>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <column>
    <property name="text">
     <string>Feature</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Value</string>
    </property>
   </column>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
